db/active_replicator_push.go

```diff
@@ -85,14 +85,19 @@ func (apr *ActivePushReplicator) _connect() error {
 		return err
 	}
 
+	singleCollection := &DatabaseCollectionWithUser{
+		DatabaseCollection: apr.config.ActiveDB.GetSingleDatabaseCollection(),
+		user:               apr.config.ActiveDB.user,
+	}
 	bh := blipHandler{
 		BlipSyncContext: apr.blipSyncContext,
 		db:              apr.config.ActiveDB,
-		collection:      apr.config.ActiveDB,
-		serialNumber:    apr.blipSyncContext.incrementSerialNumber(),
+		// FIXME: this is probably an array of collections
+		collection:   singleCollection,
+		serialNumber: apr.blipSyncContext.incrementSerialNumber(),
 	}
 
-	seq, err := apr.config.ActiveDB.ParseSequenceID(apr.Checkpointer.lastCheckpointSeq)
+	seq, err := singleCollection.ParseSequenceID(apr.Checkpointer.lastCheckpointSeq)
```

Comment:
I understand not wanting to blow up the scope of this PR, but this is probably a reasonable time to move methods off DatabaseContext/DatabaseCollection if they are clearly no longer using the context (like ParseSequenceID), since we're updating the method signature anyway.